app/ide-desktop/lib/content/src/index.ts

```diff
@@ -180,66 +180,62 @@ class Main implements AppRunner {
     }
 
     main(inputConfig?: StringConfig) {
-        contentConfig.OPTIONS.loadAll([app.urlParams()])
-        const isUsingAuthentication = contentConfig.OPTIONS.options.authentication.value
-        const isUsingNewDashboard =
-            contentConfig.OPTIONS.groups.featurePreview.options.newDashboard.value
-        const isOpeningMainEntryPoint =
-            contentConfig.OPTIONS.groups.startup.options.entry.value ===
-            contentConfig.OPTIONS.groups.startup.options.entry.default
-        const isNotOpeningProject =
-            contentConfig.OPTIONS.groups.startup.options.project.value === ''
-        if (
-            (isUsingAuthentication || isUsingNewDashboard) &&
-            isOpeningMainEntryPoint &&
-            isNotOpeningProject
-        ) {
-            const hideAuth = () => {
-                const auth = document.getElementById('dashboard')
-                const ide = document.getElementById('root')
-                if (auth) {
-                    auth.style.display = 'none'
-                }
-                if (ide) {
-                    ide.hidden = false
+        const parseOk = contentConfig.OPTIONS.loadAll([app.urlParams()])
```

Comment:
Sorry for the comment to the same place as previously. Would you be OK with changing the name to `loadAllAndDisplayHelpIfUnsuccessful`? I know it's super long, but at least it tells exactly what it does. Currently, the name can be misleading.